### PR TITLE
WD-8723 - update /stories page

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Update /stories page according to copydoc

## QA
- [demo link](https://ubuntu-com-13578.demos.haus/core/stories)
- [copy doc](https://docs.google.com/document/d/1mXnYnpaCJznDLtYn9Q74aVeEI3iFxVOMzF0pdL7wdu0/edit?usp=sharing)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc

## Issue / Card
[WD-8723](https://warthogs.atlassian.net/browse/WD-8723)

Fixes #

## Screenshots

[WD-8723]: https://warthogs.atlassian.net/browse/WD-8723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/core/stories.html [hunk 1]
```diff
@@ -140,6 +140,29 @@ <h4>IMS Evolve adopts Ubuntu Core to provide IoT-enabled business intelligence i
   <div class="row">
     <h2>Ubuntu Core in robotics</h2>
   </div>
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h4>ARX Fit builds the world’s most effective and efficient fitness machines</h4>
```
Comment (mtruj013) [inline templates/core/stories.html:145]:
```suggestion
      <h3 class="p-heading--4">ARX Fit builds the world’s most effective and efficient fitness machines</h3>
```
This breaks heading rank which isn't great for accessibility. From [w3](https://www.w3.org/WAI/tutorials/page-structure/headings/): "Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a `<h2>` is not followed directly by an `<h4>`, for example. It is ok to skip ranks when closing subsections, for instance, a `<h2>` beginning a new section, can follow an `<h4>` as it closes the previous section."

Would you mind applying the same fix throughout?

File: templates/core/stories.html [hunk 2]
```diff
@@ -217,19 +240,22 @@ <h2>Ubuntu Core in digital signage</h2>
...
+      <p>
+        <a href="/engage/screenly-empowers-security-conscious-enterprises">Screenly takes the next step in secure screen technology with Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
     </div>
     <div class="col-4 u-hide--medium u-hide--small u-align--center">
       {{
         image (
-        url="https://assets.ubuntu.com/v1/3eb6002c-screenly+box-logo-horizontal+purple+%281%29.png",
+        url="https://assets.ubuntu.com/v1/a182b1e8-Full%20Logo.png",
         alt="",
```
Comment (mtruj013) [inline templates/core/stories.html:256]:
```suggestion
        alt="Screenly logo",
```

**Review by mtruj013 — APPROVED** [review]
LGTM! Just a couple of comments

File: templates/core/stories.html (hunk 1 above)
Comment (lizzochek) [reply templates/core/stories.html]: updated, please check once again

**Review by mtruj013 — COMMENTED** [review]
LGTM 👍🏻

**Pull request merged**